README.md

```diff
@@ -1 +1,39 @@
-# angular-what-is-an-expression-readme
\ No newline at end of file
+# What is an Expression?
+
+## Overview
+
+Let's take a look at expressions. We've used expressions in our labs without even noticing it!
```

Comment:
Add a little bit more context here to hook the reader in. This paragraph should answer: what is an expression, and in a nutshell, why is it really cool?